arangod/IResearch/IResearchLink.cpp

```diff
@@ -230,7 +232,7 @@ bool IResearchLink::init(arangodb::velocypack::Slice const& definition) {
       return false;
     }
 
-    auto logicalWiew = ci->getView(vocbase.name(), std::to_string(viewId));
+    auto logicalWiew = ci->getView(vocbase.name(), viewId);
     auto* wiew = LogicalView::cast<IResearchViewDBServer>(logicalWiew.get());
```

Comment:
why "wiew" here? Is that Dutch?

Reply:
🤷‍♂️

Reply:
'wiew' is the cluster view, i.e. IResearchViewCoordinator or IResearchViewDBServer instance
'view' is an IResearchView instance...
started as a joke but proved to be useful

Reply:
Vale :smile: